src/configuration/app-containers.md

```diff
@@ -254,35 +255,35 @@ web:
             ...
 ```
 
-It has a few subkeys, which are:
+It has a few subkeys listed below:
 
 * `root`:
-    The folder from which to serve static assets for this location,
-    relative to the application root. The application root is the directory in
-    which the `.platform.app.yaml` file is located.  Typical values for this
+    The folder from which to serve static assets for this location
+    relative to the application root. The application root is the directory
+    where the `.platform.app.yaml` file is located.  Typical values for this
     property include `public` or `web`.  Setting it to `""` is not recommended,
     and its behavior may vary depending on the type of application.  Absolute
     paths are not supported.
 * `passthru`:
     Whether to forward disallowed and missing resources from this location to
-    the application. Can be true, false or an absolute URI path (with leading
-    `/`), the default value is false. For non-PHP applications it will
+    the application and can be true, false or an absolute URI path (with leading
+    `/`). The default value is false. For non-PHP applications, it will
     generally be just true or false.  In a PHP application, this will typically
     be the front controller such as `/index.php` or `/app.php`.  This entry
-    works similar to `mod_rewrite` under Apache.  Note: if the value of
-    `passthru` does not begin with the same value as the location key it is
-    under the passthru may evaluate to another entry. That may be useful when
+    works similar to `mod_rewrite` under Apache.  Note: If the value of
+    `passthru` does not begin with the same value as the location key, it is
+    under the passthru and may evaluate to another entry. That may be useful when
     you want different cache settings for different paths, for instance, but
     want missing files in all of them to map back to the same front controller.
     See the example block below.
 * `index`:
-    The file or files to consider when serving a request for a directory. Can
-    be file name, an array of file names, or *null*. Typically `index.html`.
+    The file or files to consider when serving a request for a directory and can
+    be a file name, an array of file names, or *null* (typically `index.html`).
```

Comment:
Adding a parenthetical for the typical case is fine, but it IMO it should be a self-contained sentence with capital T then. Viz, same as the original text just with parentheses.